locale/en/docs/guides/simple-profiling.md

```diff
@@ -134,15 +134,11 @@ passwords, or inside the Express framework itself.
 Since we ran our application using the --prof option, a tick file was generated
 in the same directory as your local run of the application. It should have the
 form isolate-0x124353456789-v8.log. In order to make sense of this file, we need
-to use the tick processor included in the Node.js source at
-<nodejs_dir>/tools/v8-prof/tick-processor.js. It is important that the version
-of the tick-processor that you run comes from the same version of node source as
-version of node used to generate the isolate file. This will no longer be a
-concern when the tick processor is [installed by default][]. The raw tick output
-can be processed using this tool by running:
+to use the tick processor which is bundled inside the Node.js binary. The raw tick
+output can be processed using this tool by running:
 
 ```
-node <path_to_nodejs_src>/tools/v8-prof/tick-processor.js isolate-0x101804c00-v8.log >processed.txt
+node --prof-process isolate-0x101804c00-v8.log >processed.txt
```

Comment:
Nit: We probably want to find some way to generalize the file name. I wouldn't hold this up for that, though, if that is likely to be the source of much bike-shedding.